puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/wrappers/abort_build.yaml

```diff
@@ -0,0 +1,8 @@
+- wrapper:
+    name: abort_build
```

Comment:
Keeping it would be good I think, those jobs shouldn't take that long.  Either perhaps hardcode it using the percentage type settings in this plugin or simply parameterise this wrapper macro: http://docs.openstack.org/infra/jenkins-job-builder/definition.html#macro